### PR TITLE
Remove files in plugins and themes from suggested cruft

### PR DESCRIPTION
**Pull request by mikkoaf**
Removing files from plugins and themes is generally not advisable. The cruft remover in it's current state can suggest the user such files and directories which might have unintended effects as mentioned in #101 . In this PR such items are not suggested to the user, making the cruft remover safer for the user.

### CODE REVIEW

File: lib/cruftfiles-ajax.php [hunk 1]
```diff
@@ -35,6 +35,11 @@ function list_known_cruft_dir( $name ) {
       '/data/wordpress/htdocs/wp-content/uploads/backupbuddy_backups',
       '/data/wordpress/htdocs/wp-content/updraft',
     );
+    $white_list_dirs = array(
+      "/data/wordpress/htdocs/wp-content/plugins",
```
Comment (l3ku) [inline lib/cruftfiles-ajax.php]: Please use single quotes if you are not evaluating anything in the string: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#single-and-double-quotes

File: lib/cruftfiles-ajax.php [hunk 2]
```diff
@@ -35,6 +35,11 @@ function list_known_cruft_dir( $name ) {
...
+      "/data/wordpress/htdocs/wp-content/mu-plugins",
+      "/data/wordpress/htdocs/wp-content/themes"
```
Comment (l3ku) [inline lib/cruftfiles-ajax.php]: Add trailing comma to last array item: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation

File: lib/cruftfiles-ajax.php [hunk 3]
```diff
@@ -55,6 +60,17 @@ function list_known_cruft_dir( $name ) {
         $crufts = array_merge($crufts, $cruft_found);
       }
     }
+    // This should be performed right after cruftfile search and before wp core
+    foreach ( $white_list_dirs as $dirname ) {
+      // Some directories are whitelisted and their files should not be deleted
+      $keep = array();
+      foreach ($crufts as $filename) {
```
Comment (l3ku) [inline lib/cruftfiles-ajax.php]: Add whitespace to foreach loop, see other foreach loops for reference:
```
foreach ( $crufts as $filename ) {
```

**Comment by mikkoaf** [comment]
Code style issues have been addressed.

**Pull request merged**